### ISSUE TITLE
Problem in Randomized Search CV

### ISSUE THREAD

**Issue by Nilanjan96** [edwardcooper/piidetect#1]
I used your code for Randomized Search CV for hyperparameter selection.
Then I ran this code 

search = pipe_cv.fit(df['Text'],df['Target'])

which is giving me this error. 

Parameter value is not iterable or distribution (key='logit_clf_word2vec__C', value=6.2881447134589035)

df is a dataframe that I have created using your Fake_PII text as 'text' column and labels as labels column after running it through binary_PII.

**Comment by vhemantrajudevops** [comment]
> I used your code for Randomized Search CV for hyperparameter selection.
> Then I ran this code
> 
> search = pipe_cv.fit(df['Text'],df['Target'])
> 
> which is giving me this error.
> 
> Parameter value is not iterable or distribution (key='logit_clf_word2vec__C', value=6.2881447134589035)
> 
> df is a dataframe that I have created using your Fake_PII text as 'text' column and labels as labels column after running it through binary_PII.

i am also getting this error
did you got the solution

**Comment by Nilanjan96** [comment]
> > I used your code for Randomized Search CV for hyperparameter selection.
> > Then I ran this code
> > search = pipe_cv.fit(df['Text'],df['Target'])
> > which is giving me this error.
> > Parameter value is not iterable or distribution (key='logit_clf_word2vec__C', value=6.2881447134589035)
> > df is a dataframe that I have created using your Fake_PII text as 'text' column and labels as labels column after running it through binary_PII.
> 
> i am also getting this error
> did you got the solution


I went around this. I used his model as an idea to how PII detection might work and trained my model over it. You can get his models at  https://github.com/edwardcooper/data-sentry

**Comment by edwardcooper** [comment]
Sorry for the late reply. I will look into it.
